deployment/build-and-stage.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(michaelkedar): This is intentionally separate from staging.yaml
+# to not break existing deployment process as new one is implemented
+# eventually, they can probably be combined
+
+steps:
+# TODO: Docker image creation & tagging
```

Comment:
nit: for consistency add TODO(michaelkedar).

Also, this seems pretty easy to add here in this PR. Let's just do that?

Reply:
I was going to add it when I properly consolidated the staging/production docker registries, since that will affect some other staging deployment scripts as well.